src/Reset.ts

```diff
@@ -107,48 +117,72 @@ export const resetdetails = (input: resetNames) => {
+      resetInfo.textContent = i18next.t('reset.details.acceleratorBoost', {
+        amount: format(player.prestigePoints),
+        required: format(player.acceleratorBoostCost)
+      })
       resetInfo.style.color = 'cyan'
       break
     case 'transcensionChallenge':
       currencyImage1.style.display = 'none'
-      resetCurrencyGain.textContent = '';
+      resetCurrencyGain.textContent = ''
 
-      (transcensionChallenge !== 0)? // TODO(@KhafraDev):
-        (resetInfo.style.color = 'aquamarine', resetInfo.textContent = `Are you tired of being in your Challenge or stuck? Click to leave Challenge ${transcensionChallenge}. Progress: ${format(player.coinsThisTranscension)}/${format(challengeRequirement(transcensionChallenge, player.challengecompletions[transcensionChallenge]))} Coins. TIME SPENT: ${format(player.transcendcounter)} Seconds.`):
-        (resetInfo.style.color = 'var(--crimson-text-color)', resetInfo.textContent = 'You\'re not in a Transcension Challenge right now. Get in one before you can leave it, duh!')
+      if (transcensionChallenge !== 0) {
+        resetInfo.style.color = 'aquamarine'
+        resetInfo.textContent = i18next.t('reset.details.transcensionChallenge.in', {
+          n: transcensionChallenge,
+          amount: format(player.coinsThisTranscension),
+          required: format(challengeRequirement(transcensionChallenge, player.challengecompletions[transcensionChallenge])),
+          timeSpent: format(player.transcendcounter)
+        })
+      } else {
+        resetInfo.style.color = 'var(--crimson-text-color)'
+        resetInfo.textContent = i18next.t('reset.details.transcensionChallenge.out')
+      }
       break
     case 'reincarnationChallenge':
       currencyImage1.style.display = 'none'
       resetCurrencyGain.textContent = ''
 
       if (reincarnationChallenge !== 0) {
         const goal = reincarnationChallenge >= 9 ? 'coins' : 'transcendShards'
-        const goaldesc = reincarnationChallenge >= 9 ? ' Coins' : ' Mythos Shards'
 
         resetInfo.style.color = 'silver'
-        resetInfo.textContent = `Are you done or tired of being in your Challenge? Click to leave Challenge ${reincarnationChallenge}. Progress: ${format(player[goal])}/${format(challengeRequirement(reincarnationChallenge, player.challengecompletions[reincarnationChallenge], reincarnationChallenge))} ${goaldesc}. TIME SPENT: ${format(player.reincarnationcounter)} Seconds.`
+        resetInfo.textContent = i18next.t('reset.details.reincarnationChallenge.in', {
+          n: reincarnationChallenge,
+          amount: format(player[goal]),
+          required: format(challengeRequirement(reincarnationChallenge, player.challengecompletions[reincarnationChallenge], reincarnationChallenge)),
+          requiredUnit: i18next.t('reset.requiredUnits.' + goal),
```

Comment:
This will cause incorrect translations for some languages; it needs to be split into 2 lines, 1 for coins & 1 for mythos shards.